### DISCUSSION TITLE
slimarray: gzip%E7%9A%84%E5%8E%8B%E7%BC%A9%E7%8E%87, %E5%8D%B3%E6%97%B6%E8%AE%BF%E9%97%AE - OpenACID Blog

### DISCUSSION THREAD

**Discussion by giscus** [openacid/openacid.github.io#43] [General]
# slimarray: gzip%E7%9A%84%E5%8E%8B%E7%BC%A9%E7%8E%87, %E5%8D%B3%E6%97%B6%E8%AE%BF%E9%97%AE - OpenACID Blog

slimarray 是一个静态整数压缩数组, 现实数据达到和gzip相当的压缩率, 无需解压就可以直接使用

https://blog.openacid.com/algo/slimarray/

**Comment by chanfung032** [comment]
> 例如对128个 4 bit 数的求和运算就可以通过: _mm512_reduce_add_epi64(_mm512_sad_epu8(a, _mm512_setzero_si512())) 来完成

这里是不是应该是 对 64 个 8 bit 数进行求和？
